### PR TITLE
Apply /features redesign

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Apply redesign as per [doc](https://docs.google.com/document/d/1G6FijpahZCSfxcuFAkv22wKCILEkF2u4Q0TQUfkW5i4/edit) and [design](https://www.figma.com/file/0DTIS3uNNUuDAdCKWrk8Uf/23.10---Microk8s.io?node-id=94%3A4206&mode=dev)

## QA

- View the site locally in your web browser at: https://microk8s-io-613.demos.haus/features
- Check copy and design

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-4184

### CODE REVIEW

File: templates/features/addons.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    <div class="col-6">
+      <div class="p-section--shallow">
+        <h2>Better user experience with addons</h2>
+      </div>
+      <div class="p-section--shallow">
+        <p>To keep MicroK8s light and flexible, the standard install only has the components needed to deploy Kubernetes. Depending on the use case, there are plenty of additional Kubernetes services or operators that would be nice to have.</p>
+        <p>MicroK8s addresses this with addons - extra services that can easily be added to MicroK8s. You can enable or disable addons at any time. Most are pre-configured to 'just work' without any further set up.</p>
+        <p>There are addons specifically created and supported by the MicroK8s team and a large selection of addons made by our community and partners. If your favourite is missing, you can easily make your own.</p>
+        <p>
+          <a href="https://microk8s.io/docs/addons">Check out all available addons&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/features/addons.html:13]: Should be a relative link.

File: templates/features/addons.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<section class="p-section--shallow">
+  <hr>
+  <div class="row">
+    <div class="col-6">
+      <div class="p-section--shallow">
+        <h2>Better user experience with addons</h2>
+      </div>
+      <div class="p-section--shallow">
+        <p>To keep MicroK8s light and flexible, the standard install only has the components needed to deploy Kubernetes. Depending on the use case, there are plenty of additional Kubernetes services or operators that would be nice to have.</p>
+        <p>MicroK8s addresses this with addons - extra services that can easily be added to MicroK8s. You can enable or disable addons at any time. Most are pre-configured to 'just work' without any further set up.</p>
```
Comment (anthonydillon) [inline templates/features/addons.html:10]:
```suggestion
        <p>MicroK8s addresses this with addons &mdash; extra services that can easily be added to MicroK8s. You can enable or disable addons at any time. Most are pre-configured to 'just work' without any further set up.</p>
```
Encoded mdashes are much nicer.

File: templates/features/automated-cluster-creation.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+      <div class="p-section--shallow">
+        <h2>Automated cluster creation and management</h2>
+      </div>
+      <div class="p-section--shallow">
+        <p>MicroK8s supports automated cluster creation and life cycle management with ClusterAPI (CAPI).</p>
+        <p>CAPI abstracts away the details of infrastructure provisioning, networking, and other low-level tasks, so users can define their desired cluster configuration using simple YAML manifests. </p>
+        <p>MicroK8s and CAPI allow you to easily provision clusters and deploy them behind proxies.</p>
+        <p>Rolling upgrades for high-availability (HA) clusters and worker notes mean you can upgrade clusters with no downtime. It also supports in-place upgrades for non-HA control planes.</p>
+        <p>
+          <a href="https://microk8s.io/docs/capi-provision">Learn how to deploy MicroK8s with CAPI&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/features/automated-cluster-creation.html:14]: Should be relative link

File: templates/features/custom-launch-configurations.html [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  <hr>
+  <div class="row">
+    <div class="col-6">
+      <div class="p-section--shallow">
+        <h2>Custom launch configurations</h2>
+      </div>
+      <div class="p-section--shallow">
+        <p>MicroK8s was always simple to install, but now it's simple to install, the way you want it. Use a custom Container Network Interface (CNI), set specific service options, configure registries, and enable the add-ons you want - all without an extra keystroke. Brilliant for automated, repeatable, reliable deployment and especially useful for air-gapped instals, embedded applications, and integration with public clouds.</p>
+        <p>
+          <a href="https://microk8s.io/docs/explain-launch-config">Perform a complicated custom K8s setup in seconds&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/features/custom-launch-configurations.html:11]: Relative link

File: templates/features/high-availability.html [hunk 5]
```diff
@@ -1,71 +1,34 @@
...
+          <h2>High availability (HA)</h2>
+        </div>
+        <p>HA Kubernetes clusters can withstand a failure on any component and continue serving workloads without interruption. </p>
+        <p>MicroK8s delivers a production-grade Kubernetes cluster simply by adding more MicroK8s nodes. There is no extra configuration required - install MicroK8s on three machines, run the join command to link them together and in moments, you have a production-grade Kubernetes cluster with HA enabled automatically.</p>
       </div>
-      <div class="col-3 u-hide--small u-hide--medium u-align--right">
-        <img src="https://assets.ubuntu.com/v1/9d785fad-Automatic+node+allocation.svg" width="230" alt=""/>
+      <div class="p-section--shallow">
+        <div class="u-embedded-media u-no-margin--top">
+          <iframe class="u-embedded-media__element" src="https://www.youtube.com/embed/pRVCxNSf-Oo" width="560" height="315" frameborder="0" title="What is MicroK8s?"></iframe>
```
Comment (anthonydillon) [inline templates/features/high-availability.html:14]: Looks like this video should be in a `col-9`.

File: templates/features/high-availability.html [hunk 6]
```diff
@@ -1,71 +1,34 @@
...
+    <div class="col-6">
+      <div class="p-section--shallow">
+        <h3 class="p-heading--5">Autonomous HA Kubernetes</h3>
+      </div>
+      <div class="p-section--shallow">
+        <p>Autonomy combined with high availability delivers a full Kubernetes with minimal setup, able to support mission-critical workloads with operational efficiency.</p>
+        <p>MicroK8s uses Dqlite, a high-availability SQLite, as its datastore. As soon as the cluster includes three or more nodes, Dqlite is resilient and the API services are distributed on all of them. If one node should fail or be restarted, Kubernetes keeps running and will recover itself back to full HA when the node becomes available again, with no administrative action. </p>
         <p>
-          <a href="/docs/high-availability">Read more about how HA MicroK8s works in the documentation&nbsp;&rsaquo;</a>
+          <a>Learn more about how HA MicroK8s works&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/features/high-availability.html:29]:
```suggestion
          <a href="/docs/high-availability">Learn more about how HA MicroK8s works&nbsp;&rsaquo;</a>
```

File: templates/features/nvidia-gpu-support.html [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+  <div class="row">
+    <div class="col-6">
+      <div class="p-section--shallow">
+        <h2>NVIDIA GPU support, ideal for AI/ML and HPC</h2>
+      </div>
+      <div class="p-section--shallow">
+        <p>Enable the MicroK8s GPU addon, and your Kubernetes workloads will be able to run on containers optimised for AI/ML workloads or High Performance computing (HPC) tasks.</p>
+        <p>MicroK8s uses the NVIDIA GPU Operator to support the necessary drivers, configuration and container runtimes. For the most effective use of your NVIDIA hardware, this includes support for Multi-instance GPUs (MIG).</p>
+        <p>
+          <a href="https://microk8s.io/docs/addon-gpu">Learn more about NVIDIA GPU support on MicroK8s&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/features/nvidia-gpu-support.html:12]: Relative link

**Review by anthonydillon — APPROVED** [review]
@mtruj013 LGTM a have left a few comments in the code.

**Comment by lyubomir-popov** [comment]
can we adjus he background ofhese - or make the page white if that proves difficult:
image

p-rule classes on all hrs unless there's a button or a video immediately under them. shaves off that .5rem margin-bottom

regular 4rem spacing at the bottom of the hero section, not deep:
image


strict confinement is a lower level heading I believe:
image

**Comment by mtruj013** [comment]
>can we adjus he background ofhese - or make the page white if that proves difficult:

I did this on the /resources pr, that styling will be applied here as well once that's merged 

> p-rule classes on all hrs unless there's a button or a video immediately under them. shaves off that .5rem margin-bottom

May be a good idea to add this to the [separators documentation ](https://vanillaframework.io/docs/base/separators)on Vanilla

> strict confinement is a lower level heading I believe:

This is a side nav item so I've maintained heading consistency for all of these. It wouldn't make much sense if all of these link to higher level headings, except for this one
image

hrs and regular section vs deep have been updated though if you could take another look @lyubomir-popov

**Comment by lyubomir-popov** [comment]
Thanks for the explanation, makes sense. Looks great thanks @mtruj013 +1

**Comment by juanruitina** [comment]
UX+1!

**Pull request merged**